src/internal/widgets/index.tsx

```diff
@@ -4,9 +4,10 @@ import React from 'react';
 import { getGlobalFlag } from '../utils/global-flags';
 import { useVisualRefresh } from '../hooks/use-visual-mode';
 
-export function createWidgetizedComponent<Props, Component extends React.ComponentType<Props>>(
-  Implementation: Component
-) {
+// Built-in React.FunctionComponent has always present `children` property which is not desired
+type FunctionComponent<Props> = (props: Props) => JSX.Element;
```

Comment:
This is the reason why it was not working before. For components without `children` prop (breadcrumb group and side navigation are such instances) typescript failed to resolve the type. With the new custom declaration it works fine